WordPress/src/main/java/org/wordpress/android/ui/bloggingprompts/BloggingPromptsListFragment.kt

```diff
@@ -26,31 +31,52 @@ class BloggingPromptsListFragment : ViewPagerFragment() {
 
         val section = arguments?.getSerializable(LIST_TYPE) as? PromptSection
                 ?: PromptSection.ALL
-        binding.tempText.text = section.name
-        showContent()
+        initializeViews()
 
         // DUMMY LOGIC
         showLoading()
         binding.root.postDelayed(Runnable {
             if (!isAdded) return@Runnable
             when (section) {
-                PromptSection.ALL -> showEmpty()
+                PromptSection.ALL -> showContent(
+                        listOf(
+                                generateDummyData(),
+                                generateDummyData(),
+                                generateDummyData(),
+                                generateDummyData(),
+                                generateDummyData(),
+                                generateDummyData(),
+                                generateDummyData(),
+                                generateDummyData(),
+                                generateDummyData()
+                        )
+                )
                 PromptSection.NOT_ANSWERED -> showError()
                 PromptSection.ANSWERED -> showNoConnection()
             }.exhaustive
         }, TimeUnit.SECONDS.toMillis(2))
     }
 
-    private fun showContent() {
+    private fun initializeViews() {
         with(binding) {
-            tempText.setVisible(true)
+            promptsList.layoutManager = LinearLayoutManager(activity, RecyclerView.VERTICAL, false)
```

Comment:
Would it make sense to use `apply` here? Something like:

```
promptsList.apply {
    layoutManager = ...
    addItemDecoration(...)
}
```

Reply:
Agreed. Will do.

Reply:
This is done. I've used `with()` instead of `apply` though.